### PR TITLE
Implemented Management token support

### PR DESCRIPTION
**Pull request by sunil-lakshman**
CS-42851 : Implemented Management token support.

### CODE REVIEW

File: lib/stack/index.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { Branch } from './branch'
 import { BranchAlias } from './branchAlias'
 import { AuditLog } from './auditlog'
 import { Taxonomy } from './taxonomy'
+import { ManagementToken } from './managementToken/index'
```
Comment (nadeem-cs) [inline lib/stack/index.js:22]: No need to mention index. The index file serves as root here. Just mention `'./managementToken'`

File: lib/stack/managementToken/index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+     * client.stack().managementToken().query({ query: { name: 'token_name' } })).find()
+     * .then((contentstackCollection) => console.log(contentstackCollection))
+     */
+    this.query = query({ http: http, wrapperCollection: ManagementTokenCollection })
+  }
+}
+
+export function ManagementTokenCollection (http, data) {
+  const obj = cloneDeep(data.tokens) || []
+  const managementTokenCollection = obj.map((userdata) => {
```
Comment (nadeem-cs) [inline lib/stack/managementToken/index.js:107]: Change the callback func param name here. Giving proper naming according to the value it is receiving.

File: types/stack/managementToken/index.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    acl: ACL
+}
+export interface ACL extends AnyProperty {
+    read?: boolean
+    write?: boolean
+    create?: boolean
+    update?: boolean
+}
+
+export interface Schema extends AnyProperty {
```
Comment (nadeem-cs) [inline types/stack/managementToken/index.ts:29]: Where is this schema interface used?

**Review by nadeem-cs — CHANGES_REQUESTED** [review]

File: lib/stack/managementToken/index.js (hunk 2 above)
Comment (sunil-lakshman) [reply lib/stack/managementToken/index.js]: Changed the param name.

File: types/stack/managementToken/index.ts (hunk 3 above)
Comment (sunil-lakshman) [reply types/stack/managementToken/index.ts]: schema interface has been deleted, we didn't used the schema interface anywhere.

**Review by nadeem-cs — APPROVED** [review]
Looks good @sunil-lakshman.

**Pull request merged**